7-layouts/layout.html

```diff
@@ -0,0 +1,67 @@
+<html>
+    <head>
+      <meta charset="utf-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <link rel="stylesheet" href="./css/styles.css">
+      <title>Cats</title>
+    </head>
+    <body>
+      <header>
+        <img class="logo-fire" src="./assets/flat.svg" alt="Logo of fire">
+        Grid into the future
+      </header>
+      
+      <div class="banner-container">
+        <div class="banner-img">
+          <h1 class="h1-style">Cats</h1>
+          <p class="banner-text">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed magna nibh, ultricies in risus sit amet. 
+          </p>
+        </div>
+      </div>
+      <div class="img-container">
+        
+        <div class="card-container">
+          <div class="card-info">
```

Comment:
The images are content images, please don't use a background image because it's invisible to screen readers and makes your content less accessible.